### PR TITLE
Using Bootstrap Newsweek page Milestone 1

### PR DESCRIPTION
**Pull request by jurgen1c**
#### Using Bootstrap 
In this branch we:

- Added HTML & CSS
- Added styling with Bootstrap CDN 
- Added relevant content.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1402 @@
...
+          <li class="nav-item my-2 px-3 border-right">
+            <a class="nav-link" href="#"><h4 class="bold">Vantage</h4></a>
+          </li>
+          <li class="nav-item my-2 px-3">
+            <a class="nav-link" href="#"><h4 class="bold">Weather</h4></a>
+          </li>
+        </ul>
+        <form class="form-inline my-2 my-lg-0">
+          <div class="editor-img-box">
+            <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
```
Comment (expjazz) [inline index.html]: - [x] The page is overflowing as the width diminishes.

| Yours | Original |
|:---:|:---:|
|image |image |

- [x] The page breaks in the medium screen size.

| Yours | Original |
|:---:|:---:|
| image | image |

- [x] The navbar breaks at the mobile screen size (700px)

| Yours | Original |
|:---:|:---:|
|image|image|

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1402 @@
...
+  
+            <div class="card row border-0">
+              <a href="#">
+                <div class="editor-img-box px-3">
+                  <img class="card-img-top" src="./img/donald-trump.webp" alt="trump">
+                  <div class="editor-img-title">
+                    <h5>Health</h5>
+                  </div>
+                </div>
+              </a>
```
Comment (expjazz) [inline index.html]: - [x] The design in the mobile and medium screens is completely off (font-sizes and font-weights). It is also breaking

- [x] Try to focus on the details. It needs to be close to the original.

| Yours | Original |
|:---:|:---:|
|image| image|

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1402 @@
...
+                  </h5>
+                </a>
+                <h6 class="ml-5">
+                  <strong>BY MICHAL COTLER-WUNSH</strong>
+                </h6>
+              </div>
+            </div>
+
+
+          <!--Free Sign-Up-->
```
Comment (expjazz) [inline index.html:608]: - [x] You missed a section here. (don't mind the navbar in the screenshot)

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1402 @@
...
+
+    <section class="row mx-auto d-block container-fluid">
+      <div class="row">
+        <div class="col-12 d-inline-flex justify-content-center m-sm-3 border-bottom py-3">
+          <img class="float-left m-1 mr-3" src="./img/N-icon.png" alt="N icon" width="30">
+          <h1>
+            In The Magazine
+          </h1>
+        </div>
+      </div>
```
Comment (expjazz) [inline index.html:1126]: - [x] The height and alignment of the images are too different. Try to make closer to the original.


| Yours | Original |
|:---:|:---:|
|image| image |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1402 @@
...
+      <div class="row">
+        <div class="col-lg-4 col-md-12 card border-0">
+          <a href="">
+            <img class="card-img-top" src="./img/us-president-donald-trump.webp" alt="us p">
+          </a>
+          <div class="card-body">
+            <a href="">
+              <h2 class="card-title">Trump's Coronavirus, Protest and Russia Ratings Plummet to 30s</h2>
```
Comment (expjazz) [inline index.html]: - [x] Fonts are off here as well

| Yours | Original |
|:---:|:---:|
|image | image |

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1402 @@
...
+            </div>
+          </div>
+        </div>
+        <div class="col-12 col-md-4 my-4">
+          <h3>NEWSLETTER</h3>
+          <div class="d-inline-flex h-100">
+            <div class="mr-2">
+              <img width="200" height="200" src="./img/130387482-newsletter-subscription-button-on-mobile-phone-screen-business-marketing-concept-.jpg" alt="">
+            </div>
+            <div>
```
Comment (expjazz) [inline index.html]: - [x] In the small screen size, it does not have an image.

| Yours | Original |
|:---:|:---:|
|image |image |

- [x] IT is breaking at 1022 px, and overflowing

| Yours | Original |
|:---:|:---:|
| image | image |

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1402 @@
...
+        <p>&copy; 2020 NEWSWEEK</p>
+      </div>
+      <div class="col-md-5 col-12 d-flex justify-content-md-end">
+          <div class="mr-2">
+            <i class="fab fa-facebook-square fa-2x"></i>
+          </div>
+          <div class="mr-2">
+            <i class="fab fa-twitter fa-2x"></i>
+          </div>
+          <div class="mr-2">
```
Comment (expjazz) [inline index.html]: - [x] copyright alignment on the mobile screen is different. Also, pay attention to the font styling.

| Yours | Original |
|:---:|:---:|
|image | image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

#### Hey @jurgen1c and @Ceci007 

You did a very good job on this project. The mobile screen size looks quite nice, and the large screen also looks great.

However, in order to achieve mastery, there are a few aspects to improve:

- [x] I do not see the layout for the tablet screen size. You are missing the design for a whole screen size. Kindly address that.

| Yours | Original |
|:---:|:---:|
|image |image|

- [x] The general fonts are off. Try to make more like the original.

- [x] Check the inline comments below.

- [x] The order on the mobile screen size is wrong. 
#### It should be: Top Story => Featured Stories => The Debate => Opinion => More Stories => My Turn => In the magazine. 
Yours is: Top Story => My Turn => More Stories => The debate => Opinion => Featured Stories => In the magazine.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

File: index.html (hunk 3 above)
Comment (jurgen1c) [reply index.html:619]: Hello,
This is an Ad as you can see from the title says sponsored insight and it doesn't show up with an add blocker. we thought we shouldn't include ads or should we?

**Comment by jurgen1c** [comment]
newsweek no ads
 This is how the page looks to us without any ads adding screenshot as requested by previous reviewer.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,1379 @@
...
+        <div class="col-12 col-lg-5 my-5 py-2">
+          <h3 class="red-title">NEWSLETTER</h3>
+          <div class="d-inline-flex h-100">
+            <div class="mr-2 d-none d-md-block">
+              <img class="h-100" width="200" src="./img/130387482-newsletter-subscription-button-on-mobile-phone-screen-business-marketing-concept-.jpg" alt="">
+            </div>
+            <div>
+              <h4>The Starting 5</h4>
+              <p>
+                See why nearly a quarter of a million subscribers begin their day with the Starting 5.
```
Comment (expjazz) [inline index.html]: - [x] This section looks much better. You just need to just the SUBSCRIBE to be equal (the layout), put some border on THE STARTING FIVE, and centralize both.

| Yours | Original |
|:---:|:---:|
|image |image  |

File: index.html [hunk 9]
```diff
@@ -0,0 +1,1379 @@
...
+      <ul class="navbar-nav mr-auto buttons-menus font-weight-bold">
+          <li class="nav-item my-2">
+            <a href="#" class="border-right px-2 py-1 nav-link">U.S</a>
+          </li>
+          <li class="nav-item my-2">
+              <a href="#" class="border-right px-2 py-1 nav-link">Word</a>
+          </li>
+          <li class="nav-item my-2">
+              <a href="#" class="border-right px-2 py-1 nav-link">Business</a>
+          </li>
```
Comment (expjazz) [inline index.html]: - [x] This section needs more width.

| Yours | Original |
|:---:|:---:|
|image |image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes required :recycle: 

You did a very good job working on the changes I requested, and we are almost done. I saw two more details that came after what you changed, but they are minor and it won't take much time.

#### However, there is one problem: The order of the sections in the mobile version is wrong. Kindly address that (you need to make it in the same as the original; To do so, you might need to use extra HTML).

Again, your project look really great.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

You did a fine job on this project :100: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**